package.json

```diff
@@ -52,14 +53,13 @@
     "enzyme-to-json": "^3.2.2",
     "gh-pages": "^1.0.0",
     "jest": "^21.2.1",
-    "material-ui": "^1.0.0-beta.30",
     "react": "^16.1.1",
```

Comment:
React (and ReactDOM) should be updated to `^16.3.0`, since Material-UI requires at least 16.3.0 as of rc.0